src/mainApp/AppHolder.js

```diff
@@ -52,7 +53,9 @@ const AppHolder = () => {
   return (
     <ActionSheetProvider>
       <SimpleStore.Container>
-        <AppHot />
+        <GDStore.Container>
```

Comment:
gdstore isnt required on signup router that's why it was only used in the router.
if it has nothing to do with the bug please restore

Reply:
yes it's the bug fix. the store container (which is actually a React Context provider should always be present and placed as much on top as it's possible)

Reply:
> > What was the source of the bug?
> 
> `Component "n" does not seem to be nested in an Undux <Container>.`
> `Store failed with exception: Cannot read property \'get\' of null', [TypeError: Cannot read property 'get' of null]`
> 
> GD store's container is removing from the JSX in some cases, the store also destroys. App is trying to render something / access store and crashes
> 
> This happens because GDStore.Container is located under Router component which is lazy loaded.
> 
> I moved it to the app holder with the SimpleStore's container

hmm i'm not aware that context needs to be defined at the top, it should be definable everywhere in the component hierarchy.
did you double check that this is in did the cause of the bug?

Reply:
During the tests i found which components is failing to render, the issue actually relates to the simple store. So I restored this.
But the reason is the same - store destroyed, connect removed from JSX. 
Found a clue - we're re creating the store during initStore() call. I'm updating app to avoid this
Have solution prototype for web but it breaks app bootstrap on native. Fixing this right now